airflow-core/src/airflow/utils/__init__.py

```diff
@@ -46,5 +46,11 @@ def __getattr__(name: str):
     "xcom": {
         "XCOM_RETURN_KEY": "airflow.models.xcom.XCOM_RETURN_KEY",
     },
+    "weight_rule": {
+        "WeightRule": "airflow.sdk.definitions.weight_rule.WeightRule",
```

Comment:
WeightRule is used in config + sdk.

This one could probably just be duplicated.